book/src/keymap.md

```diff
@@ -335,32 +335,22 @@ experience.
 | `Backspace`, `Ctrl-h`                       | Delete previous char        | `delete_char_backward`  |
 | `Delete`, `Ctrl-d`                          | Delete next char            | `delete_char_forward`   |
 
-However, if you really want navigation in insert mode, this is supported. An
-example config that gives the ability to use arrow keys while still in insert
-mode:
-
-```toml
-[keys.insert]
-"up" = "move_line_up"
-"down" = "move_line_down"
-"left" = "move_char_left"
-"right" = "move_char_right"
-"C-b" = "move_char_left"
-"C-f" = "move_char_right"
-"A-b" = "move_prev_word_end"
-"C-left" = "move_prev_word_end"
-"A-f" = "move_next_word_start"
-"C-right" = "move_next_word_start"
-"A-<" = "goto_file_start"
-"A->" = "goto_file_end"
-"pageup" = "page_up"
-"pagedown" = "page_down"
-"home" = "goto_line_start"
-"C-a" = "goto_line_start"
-"end" = "goto_line_end_newline"
-"C-e" = "goto_line_end_newline"
-"A-left" = "goto_line_start"
-```
+### Insert Mode (beginners)
+
+These keys are not recommended, it is for those who do not read documentations.
```

Comment:
I am not sure if I even need this line given that most likely they won't even read documentation.

Reply:
This should say "These keys are not recommended, but are included for new users less familiar with modal editors"

Reply:
Genuinely happy to see this. Again, not for me (I’ve already added the mappings back in my config and am running from source) but in terms of the culture of the project and what it says for how new folks are welcomed. This might seem like a tiny thing but it does make a world of difference in terms of how someone feels when they’re not familiar with a new tool.

This is saying “hey, we recognise this is how nearly everything out there works and that you have muscle memory you’re bringing with you… that’s ok. This tool is designed with a different workflow in mind – which you’ll learn in time unless we lose you in the first 30 seconds – but we will cater for your needs to make your introduction more seamless.”

Happy to see Helix embracing a more open/welcoming approach than “our way or the highway.” I truly believe that (whether by accident or not), this is one of those things that has brought a lot of people who would otherwise not have used a modal editor to Helix.

:two_hearts: